README.md

```diff
@@ -68,7 +68,7 @@ To avoid needing to read the entire transaction log every time a table is loaded
 
 Delta Lake ACID guarantees are predicated on the atomicity and durability guarantees of the storage system. Specifically, we require the storage system to provide the following. 
 
-1. **Atomic visibility**: There must a way for a file to visible in its entirely or not visible at all. 
+1. **Atomic visibility**: There must be a way for a file to visible in its entirely or not visible at all. 
```

Comment:
Might as well fix the whole sentance. Should be the following (**bolded** for emphasis on new changes)

"There must be a way for a file to **be** visible in its **entirety** or not visible at all. "